packages/pds/src/util/http.ts

```diff
@@ -0,0 +1,15 @@
+import { ServerResponse } from 'node:http'
+
+export function appendVary(res: ServerResponse, value: string) {
+  const current = res.getHeader('Vary')
+  if (current == null || typeof current === 'number') {
+    res.setHeader('Vary', value)
+  } else {
+    const alreadyIncluded = Array.isArray(current)
+      ? current.some((value) => value.includes(value))
+      : current.includes(value)
```

Comment:
Due to the way inclusion is checked as a substring, I think this usage wont have the expected result:
```ts
appendVary(res, 'atproto-accept-labelers') // Vary: atproto-accept-labelers
appendVary(res, 'accept')                  // Vary: atproto-accept-labelers
```